### PR TITLE
description of triggers

### PR DESCRIPTION
**Pull request by Vegebutcher**

### CODE REVIEW

File: g1ant-language/triggering/README.md [hunk 1]
```diff
@@ -18,20 +20,24 @@ Note that some possible arguments are not required to be set because they alread
...
-Arguments generated while executing the script. In order to get them, use special variable `♥task` and inside these special rectangle characters `⟦⟧` insert the name of an argument.
-e.g. `♥task⟦filepath⟧`
+Task arguments are generated while executing the script. In order to get them, use a special variable `♥task` and insert the name of an argument inside special double brackets `⟦⟧`,
+e.g. `♥task⟦filepath⟧`.
 
-For the list of possible task or initial arguments, please visit manual page regarding a specific trigger.
+For the list of possible task or initial arguments, see the description of a specific trigger.
 
-Note that triggers are not active at the start of G1ANT.Robot. Whenever we want to use them, the triggersactive option must be switched on before launching the script. You can do this by choosing `Triggers/Active` from the Menu in Robot.
+Note that triggers are not active when G1ANT.Robot starts. Whenever you want to use them, you have to activate them before launching the script. You can do this by choosing `Triggers/Active` menu in Robot window.
```
Comment (WiktoriaPrusik) [inline g1ant-language/triggering/README.md]: in developer version triggers are not active when G1ANT.Robot starts but in production version they are active - you need to update this info

it's not true that "Whenever you want to use them, you have to activate them before launching the script." once you activate them, they will be active until you close G1ANT.Robot

File: g1ant-language/triggering/mail-trigger.md [hunk 2]
```diff
@@ -2,51 +2,51 @@
...
-**Example of defining File Trigger in Settings**
+| -------- | ---- |
+| `Title` | The title of an incoming mail |
+| `AttachmentNames` | A list of attachments in an incoming mail |
+| `Content` | Incoming mail content (text) |
+| `From` | Sender's email address |
+| `To` | Recipient's email address |
+| `HtmlBody` | Incoming mail content (html) |
+| `Uid` | A unique identifier of a mail given by a server |
+| `Date` | Sent date |
```
Comment (WiktoriaPrusik) [inline g1ant-language/triggering/mail-trigger.md]: date sent

**Review by WiktoriaPrusik — CHANGES_REQUESTED** [review]

File: g1ant-language/triggering/README.md (hunk 1 above)
Comment (Vegebutcher) [reply g1ant-language/triggering/README.md]: OK, your remarks included.

File: g1ant-language/triggering/mail-trigger.md (hunk 2 above)
Comment (Vegebutcher) [reply g1ant-language/triggering/mail-trigger.md]: checked against reality: it's like that :D

**Review by WiktoriaPrusik — APPROVED** [review]
everything just perfect!

**Pull request merged**